website/sidebars-full.json

```diff
@@ -1,123 +1,70 @@
 {
   "docs": {
-    "Overview": ["index", "getting-started", "glossary"],
-    "Conceptual": [
-      "conceptual/index",
-      {
-        "type": "subcategory",
-        "label": "Substrate Core",
-        "ids": [
-          "conceptual/core/codec",
-          "conceptual/core/consensus",
-          "conceptual/core/executor",
-          "conceptual/core/networking",
-          "conceptual/core/off-chain-workers",
-          "conceptual/core/storage"
-        ]
-      },
-      {
-        "type": "subcategory",
-        "label": "Substrate Nodes",
-        "ids": [
-          "conceptual/node/client",
-          "conceptual/node/light-client",
-          "conceptual/node/extrinsics",
-          "conceptual/node/rpc",
-          "conceptual/node/tx-pool"
-        ]
-      },
-      {
-        "type": "subcategory",
-        "label": "Cryptography",
-        "ids": [
-          "conceptual/cryptography/index",
-          "conceptual/cryptography/keys",
-          "conceptual/cryptography/session-keys",
-          "conceptual/cryptography/ss58-address-format"
-        ]
-      },
-      {
-        "type": "subcategory",
-        "label": "Substrate Runtime",
-        "ids": [
-          "conceptual/runtime/index",
-          "conceptual/runtime/frame",
-          "conceptual/runtime/weight",
-          "conceptual/runtime/primitives"
-        ]
-      },
-      {
-        "type": "subcategory",
-        "label": "Smart Contracts",
-        "ids": [
-          "conceptual/runtime/contracts/index",
-          "conceptual/runtime/contracts/contracts_module",
-          "conceptual/runtime/contracts/evm_module",
-          "conceptual/runtime/contracts/ink"
-        ]
-      }
+    "Overview": [
+      "index",
+      "getting-started",
+      "conceptual/runtime/frame",
+      "glossary"
+    ],
+    "Substrate Core": [
+      "conceptual/core/codec",
+      "conceptual/core/consensus",
+      "conceptual/core/executor",
+      "conceptual/core/networking",
+      "conceptual/core/off-chain-workers",
+      "conceptual/core/storage"
+    ],
+    "Substrate Nodes": [
```

Comment:
Why not call it `Substrate Client`. Then we can add the `client` part as an overview.
The section will explain the different types of clients, their parts and how to interact with them.